build/yamls/flow-aggregator.yml

```diff
@@ -131,11 +131,20 @@ data:
     # If no L4 transport proto is given, we consider tcp as default.
     #externalFlowCollectorAddr: ""
 
-    # Provide flow export interval as a duration string. This determines how often the flow aggregator exports flow
-    # records to the flow collector.
-    # Flow export interval should be greater than or equal to 1s (one second).
-    # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
-    #flowExportInterval: 60s
+    # Provide the active flow record timeout as a duration string. This determines
+    # how often the flow aggregator exports the active flow records to the flow
+    # collector. Thus, for flows with a continuous stream of packets, a flow record
+    # will be exported to the collector once the elapsed time since the last export
+    # event is equal to the value of this timeout.
+    # Defaults to "60s". Follow the time units of duration type.
+    #activeFlowRecordTimeout: 60s
+
+    # Provide the inactive flow record timeout as a duration string. This determines
+    # how often the flow aggregator exports the inactive flow records to the flow
+    # collector. A flow record is considered to be inactive if no matching record
+    # has been received from any Node in the cluster since the last export event.
+    # Defaults to "120s". Follow the time units of duration type.
+    #inactiveFlowRecordTimeout: 120s
```

Comment:
IIRC we typically do not repeat the default values ion the comment since we already give them in the commented-out config parameter definition.
Also, it seems we typically use `# Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".` instead of `Follow the time units of duration type.`. Why the change here?

Reply:
We can stick with the model of listing all units, did not realize I am taking the old text as the reference.